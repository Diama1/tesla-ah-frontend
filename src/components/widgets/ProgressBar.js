import React from 'react';
import { ProgressBar } from 'react-step-progress-bar';
import 'react-step-progress-bar/styles.css';

const progressBar = ({ onPercentageHandle }) => (
  <div>
    <ProgressBar
      percent={onPercentageHandle}
      filledBackground="linear-gradient(to right, #fefb72, #f0bb31)"
    />
  </div>
);

export default progressBar;
