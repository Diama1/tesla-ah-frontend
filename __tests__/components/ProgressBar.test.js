import React from 'react';
import { shallow } from 'enzyme';
import ProgressBar from '../../src/components/widgets/ProgressBar';

describe('ProgressBar Tests', () => {
  it('should render the ProgressBar component', () => {
    const props = {
      onPercentageHandle: jest.fn(),
    };
    const wrapper = shallow(<ProgressBar {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
