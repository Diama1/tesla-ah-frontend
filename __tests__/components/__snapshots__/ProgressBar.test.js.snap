// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar Tests should render the ProgressBar component 1`] = `ShallowWrapper {}`;
